grammars/php.cson

```diff
@@ -3624,6 +3627,56 @@
         ]
       }
     ]
+  'match_statement':
+    'patterns': [
+      {
+        'match': '\\s+(?=match\\b)'
```

Comment:
I've copied this from switch, but I really don't know why this is present and commit history didn't help with solving this mystery. Anyone?